cabby/geo/map_processing/map_processor.py

```diff
@@ -39,6 +39,8 @@ def main(argv):
     del argv  # Unused.
     map = map_structure.Map(FLAGS.region, FLAGS.level)
     print(map.poi)
+    map.poi.to_csv(FLAGS.region+'_poi.csv') 
+    map.streets.to_csv(FLAGS.region+'_streets.csv')
```

Comment:
Please add a flag that allows the caller to specify an output directory. It should also check whether the files exist already and refuse to write them if so.

Reply:
done